src/helm/benchmark/static/schema_lite_v2.yaml

```diff
@@ -103,6 +108,11 @@ metrics:
     short_display_name: WB Score
     description: Score of the AI output judged by GPT-4o.
     lower_is_better: false
+  - name: wildbench_score
```

Comment:
should this be _rescaled?

Reply:
Yes thanks for catching. This pull request was missing a commit. Should be fixed now.